msg/RTKStatus.msg

```diff
@@ -1,12 +1,36 @@
+uint8   MODEM_STATE_OFF = 0
```

Comment:
A comment at the beginning of the block that explains that these should be used with `dongle_modem_state` would be helpful. Something like:

```python
# Possible values for dongle_modem_state
```

Might also be useful to add a comment to `dongle_modem_state` that points users to `MODEM_STATE_*` as the possible values for that field